saiku-webapp/src/main/resources/ccrs-mondrian-cp3.xml

```diff
@@ -36,35 +36,53 @@
+                            CASE WHEN cp3.HASUNDAF = TRUE THEN CASE WHEN cp3undaf.IS_DEVELOPING_NEWUNDAF = TRUE THEN 1 ELSE CASE WHEN cp3undaf.IS_DEVELOPING_NEWUNDAF = FALSE THEN 0 ELSE -1 END END ELSE -1 END as IS_DEVELOPING_NEWUNDAF,
+                            CASE WHEN cp3.HAS_COMMON_COUNTRY_ASSESSMENT = TRUE THEN 1 ELSE 0 END as HAS_COMMON_COUNTRY_ASSESSMENT_COUNT,
+                            CASE WHEN cp3.HAS_COMMON_COUNTRY_ASSESSMENT = TRUE THEN 1 ELSE CASE WHEN cp3.HAS_COMMON_COUNTRY_ASSESSMENT = FALSE THEN 0 ELSE -1 END END as HAS_COMMON_COUNTRY_ASSESSMENT,
                             CASE WHEN cp3.HAS_COMMON_COUNTRY_ASSESSMENT = TRUE THEN cp3assessment.ASSESSMENT_YEAR ELSE null END as ASSESSMENT_YEAR,
-                            CASE cp3.HAS_DISAGGREGATED_DATA WHEN TRUE THEN 1 WHEN FALSE THEN 0 ELSE null END as HAS_DISAGGREGATED_DATA,
+                            CASE WHEN cp3.HAS_COMMON_COUNTRY_ASSESSMENT = TRUE THEN CASE WHEN cp3assessment.HAS_DISAGGREGATED_DATA = TRUE THEN 1 ELSE 0 END ELSE null END as HAS_DISAGGREGATED_DATA_COUNT,
+                            CASE WHEN cp3.HAS_COMMON_COUNTRY_ASSESSMENT = TRUE THEN CASE WHEN cp3assessment.HAS_DISAGGREGATED_DATA = TRUE THEN 1 ELSE CASE WHEN cp3assessment.HAS_DISAGGREGATED_DATA = FALSE THEN 0 ELSE -1 END END ELSE -1 END as HAS_DISAGGREGATED_DATA,
                             CASE WHEN cp3.HAS_COMMON_COUNTRY_ASSESSMENT = FALSE THEN cp3assessment.OTHER_ANALYSIS_SOURCE_ID ELSE null END as OTHER_ANALYSIS_SOURCE_ID,
-                            CASE WHEN cp3.HAS_NATIONAL_DEVELOPMENT_PLAN = TRUE THEN 1 ELSE 0 END as HAS_NATIONAL_DEVELOPMENT_PLAN,
-                            CASE WHEN cp3.HAS_NATIONAL_DEVELOPMENT_PLAN = TRUE THEN CASE WHEN cp3plan.HASUNSUPPORT = TRUE THEN 1 ELSE 0 END ELSE null END as HASUNSUPPORT,
-                            CASE WHEN cp3.HAS_JOINT_PROGRAMMES = TRUE THEN 1 ELSE 0 END as HAS_JOINT_PROGRAMMES,
-                            CASE WHEN cp3.HASUNDAFANNUAL_REVIEW = TRUE THEN 1 ELSE 0 END as HASUNDAFANNUAL_REVIEW,
+                            CASE WHEN cp3.HAS_NATIONAL_DEVELOPMENT_PLAN = TRUE THEN 1 ELSE 0 END as HAS_NATIONAL_DEVELOPMENT_PLAN_COUNT,
+                            CASE WHEN cp3.HAS_NATIONAL_DEVELOPMENT_PLAN = TRUE THEN 1 ELSE CASE WHEN cp3.HAS_NATIONAL_DEVELOPMENT_PLAN = FALSE THEN 0 ELSE -1 END END as HAS_NATIONAL_DEVELOPMENT_PLAN,
+                            CASE WHEN cp3.HAS_NATIONAL_DEVELOPMENT_PLAN = TRUE THEN CASE WHEN cp3plan.HASUNSUPPORT = TRUE THEN 1 ELSE 0 END ELSE null END as HASUNSUPPORT_COUNT,
+                            CASE WHEN cp3.HAS_NATIONAL_DEVELOPMENT_PLAN = TRUE THEN CASE WHEN cp3plan.HASUNSUPPORT = TRUE THEN 1 ELSE CASE WHEN cp3plan.HASUNSUPPORT = FALSE THEN 0 ELSE -1 END END ELSE -1 END as HASUNSUPPORT,
+                            CASE WHEN cp3.HASMDGACCELERATION_FRAMEWORK = TRUE THEN 1 ELSE 0 END as HASMDGACCELERATION_FRAMEWORK_COUNT,
```

Comment:
HASMDGACCELERATION_FRAMEWORK was removed in develop.

Reply:
It is in line 59, was not changed nor removed